tools/serverpod_cli/lib/src/logger/logger.dart

```diff
@@ -47,11 +47,11 @@ abstract class Logger {
 
   /// Display a [SourceSpanSeverityException] to the user.
   /// Commands should use this to log [SourceSpanSeverityException] with
-  /// enhanced highlighting where possible.
+  /// enhanced highlighting if possible.
   void sourceSpanSeverityException(
-    SourceSpanSeverityException sourceSpan,
-    LogLevel logLevel,
-  );
+    SourceSpanSeverityException sourceSpan, {
```

Comment:
Question: I opted to make the logger method specific to the Type that we have special formatting for. That makes it very clear to the caller that only these type of objects have special formatting instead of allowing all SourceSpanExceptions.

I think this makes it more clear to any further logger implementation what logging types need to be supported.

Reply:
Let's make this `sourceSpanException` and make a special case for `SourceSpanSeverityException`. Handle `SourceSpanException` as error.